packages/internal-paths/package.json

```
@@ -0,0 +1,73 @@
+{
+  "name": "@redwoodjs/internal-paths",
+  "version": "3.2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/redwoodjs/redwood.git",
+    "directory": "packages/internal-paths"
+  },
+  "license": "MIT",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn build:js && yarn build:types",
+    "build:clean-dist": "rimraf 'dist/**/*/__tests__'",
+    "build:js": "babel src -d dist --extensions \".js,.ts,.tsx\" --copy-files --no-copy-ignored && yarn build:clean-dist",
+    "build:types": "tsc --build --verbose",
+    "build:watch": "nodemon --watch src --ext \"js,ts,tsx\" --ignore dist --exec \"yarn build\"",
+    "fix:permissions": "chmod +x dist/generate/generate.js dist/generate/watch.js",
+    "prepublishOnly": "NODE_ENV=production yarn build",
+    "test": "jest src",
+    "test:watch": "yarn test --watch"
+  },
+  "dependencies": {
+    "@babel/parser": "7.20.3",
+    "@babel/plugin-transform-typescript": "7.20.2",
+    "@babel/register": "7.18.9",
+    "@babel/runtime-corejs3": "7.20.1",
+    "@babel/traverse": "7.20.1",
+    "@graphql-codegen/add": "3.2.1",
+    "@graphql-codegen/cli": "2.13.7",
+    "@graphql-codegen/core": "2.6.2",
+    "@graphql-codegen/schema-ast": "2.5.1",
+    "@graphql-codegen/typescript": "2.7.4",
+    "@graphql-codegen/typescript-operations": "2.5.4",
+    "@graphql-codegen/typescript-react-apollo": "3.3.4",
+    "@graphql-codegen/typescript-resolvers": "2.7.4",
+    "@iarna/toml": "2.2.5",
+    "@redwoodjs/graphql-server": "3.2.0",
+    "babel-plugin-graphql-tag": "3.3.0",
+    "babel-plugin-polyfill-corejs3": "0.6.0",
+    "chalk": "4.1.2",
+    "core-js": "3.26.0",
+    "deepmerge": "4.2.2",
+    "esbuild": "0.15.13",
+    "fast-glob": "3.2.12",
+    "findup-sync": "5.0.0",
+    "fs-extra": "10.1.0",
+    "graphql": "16.6.0",
+    "kill-port": "1.6.1",
+    "prettier": "2.7.1",
+    "rimraf": "3.0.2",
+    "string-env-interpolation": "1.0.1",
+    "systeminformation": "5.12.14",
+    "terminal-link": "2.1.1",
+    "typescript": "4.7.4"
+  },
```

Based on dist, this is all we need in dependencies. (Depending on which of the tests in `__tests__` we keep, we may need to move some of the `dependencies` I removed to `devDependencies`):

```suggestion
  "dependencies": {
    "@babel/runtime-corejs3": "7.20.1",
    "@iarna/toml": "2.2.5",
    "core-js": "3.26.0",
    "deepmerge": "4.2.2",
    "fast-glob": "3.2.12",
    "findup-sync": "5.0.0",
    "string-env-interpolation": "1.0.1"
  },
```

---

I just did a search for what packages were used in the tests and kept those as devDeps